new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -485,9 +580,16 @@ public static boolean deleteNamespace(String name) {
     KubernetesApiResponse<V1Namespace> response = namespaceClient.delete(name);
 
     if (!response.isSuccess()) {
```

Comment:
does this work? does it actually wait for the delete to complete and then return this? 
also - this needs to be wrapped in a thread and run with a timeout -- you can create a JIRA to add that later, if you want to do it in a separate PR.
as written, this will fail if a finalizer prevents the ns deletion from completing